File: app/src/main/java/com/brainwellnessspa/DashboardOldModule/Models/SessionListModel.java
package com.brainwellnessspa.DashboardOldModule.Models;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.List;

public class SessionListModel {
    @SerializedName("ResponseData")
    @Expose
    private List<ResponseData> responseData = null;
    @SerializedName("ResponseCode")
    @Expose
    private String responseCode;
    @SerializedName("ResponseMessage")
    @Expose
    private String responseMessage;
    @SerializedName("ResponseStatus")
    @Expose
    private String responseStatus;

    public List<ResponseData> getResponseData() {
        return responseData;
    }

    public void setResponseData(List<ResponseData> responseData) {
        this.responseData = responseData;
    }

    public String getResponseCode() {
        return responseCode;
    }

    public void setResponseCode(String responseCode) {
        this.responseCode = responseCode;
    }

    public String getResponseMessage() {
        return responseMessage;
    }

    public void setResponseMessage(String responseMessage) {
        this.responseMessage = responseMessage;
    }

    public String getResponseStatus() {
        return responseStatus;
    }

    public void setResponseStatus(String responseStatus) {
        this.responseStatus = responseStatus;
    }

    public class ResponseData {
        @SerializedName("Id")
        @Expose
        private String id;
        @SerializedName("Name")
        @Expose
        private String name;
        @SerializedName("CatName")
        @Expose
        private String catName;
        @SerializedName("Image")
        @Expose
        private String image;
        @SerializedName("Category")
        @Expose
        private String category;
        @SerializedName("CatMenual")
        @Expose
        private String catMenual;
        @SerializedName("DescInfusion")
        @Expose
        private String descInfusion;
        @SerializedName("Desc")
        @Expose
        private String desc;
        @SerializedName("Date")
        @Expose
        private String date;
        @SerializedName("Duration")
        @Expose
        private String duration;
        @SerializedName("Time")
        @Expose
        private String time;
        @SerializedName("Status")
        @Expose
        private String status;
        @SerializedName("Session")
        @Expose
        private String session;

        public String getCatName() {
            return catName;
        }

        public void setCatName(String catName) {
            this.catName = catName;
        }

        public String getId() {
            return id;
        }

        public void setId(String id) {
            this.id = id;
        }


        public String getName() {
            return name;
        }

        public void setName(String name) {
            this.name = name;
        }

        public String getCategory() {
            return category;
        }

        public void setCategory(String category) {
            this.category = category;
        }

        public String getCatMenual() {
            return catMenual;
        }

        public void setCatMenual(String catMenual) {
            this.catMenual = catMenual;
        }

        public String getDescInfusion() {
            return descInfusion;
        }

        public void setDescInfusion(String descInfusion) {
            this.descInfusion = descInfusion;
        }

        public String getDesc() {
            return desc;
        }

        public void setDesc(String desc) {
            this.desc = desc;
        }

        public String getImage() {
            return image;
        }

        public void setImage(String image) {
            this.image = image;
        }

        public String getDate() {
            return date;
        }

        public void setDate(String date) {
            this.date = date;
        }

        public String getDuration() {
            return duration;
        }

        public void setDuration(String duration) {
            this.duration = duration;
        }

        public String getTime() {
            return time;
        }

        public void setTime(String time) {
            this.time = time;
        }

        public String getStatus() {
            return status;
        }

        public void setStatus(String status) {
            this.status = status;
        }

        public String getSession() {
            return session;
        }

        public void setSession(String session) {
            this.session = session;
        }
    }

}

File: app/src/main/java/com/brainwellnessspa/dashboardOldModule/models/SessionListModel.kt
package com.brainwellnessspa.dashboardOldModule.models

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class SessionListModel {
    @SerializedName("ResponseData")
    @Expose
    var responseData: List<ResponseData>? = null

    @SerializedName("ResponseCode")
    @Expose
    var responseCode: String? = null

    @SerializedName("ResponseMessage")
    @Expose
    var responseMessage: String? = null

    @SerializedName("ResponseStatus")
    @Expose
    var responseStatus: String? = null

    inner class ResponseData {
        @SerializedName("Id")
        @Expose
        var id: String? = null

        @SerializedName("Name")
        @Expose
        var name: String? = null

        @SerializedName("CatName")
        @Expose
        var catName: String? = null

        @SerializedName("Image")
        @Expose
        var image: String? = null

        @SerializedName("Category")
        @Expose
        var category: String? = null

        @SerializedName("CatMenual")
        @Expose
        var catMenual: String? = null

        @SerializedName("DescInfusion")
        @Expose
        var descInfusion: String? = null

        @SerializedName("Desc")
        @Expose
        var desc: String? = null

        @SerializedName("Date")
        @Expose
        var date: String? = null

        @SerializedName("Duration")
        @Expose
        var duration: String? = null

        @SerializedName("Time")
        @Expose
        var time: String? = null

        @SerializedName("Status")
        @Expose
        var status: String? = null

        @SerializedName("Session")
        @Expose
        var session: String? = null
    }
}